src/Makefile

```diff
@@ -213,7 +213,19 @@ $(MONO_PATH)/mcs/class/lib/monotouch/reference_Facades/System.Drawing.Primitives
 $(IOS_BUILD_DIR)/%/Facades/System.Drawing.Primitives.dll: $(MONO_PATH)/mcs/class/lib/monotouch/%_Facades/System.Drawing.Primitives.dll | $(IOS_BUILD_DIR)/compat/Facades $(IOS_BUILD_DIR)/reference/Facades
 	$(Q) cp $< $@
 
+$(MONO_PATH)/mcs/class/lib/monotouch/compat_Facades/netstandard.dll: $(IOS_BUILD_DIR)/compat/monotouch.dll $(IOS_BUILD_DIR)/compat/OpenTK-1.0.dll
```

Comment:
We've removed support for XI/Classic, so you don't need to build new assemblies for Classic.